src/components/shared/qrcode.tsx

```diff
@@ -31,9 +31,11 @@ export function QRCode({ text, options = {}, canvasProps }: QRProps) {
     useEffect(() => {
         if (!ref.current) return
 
-        qr.toCanvas(ref.current, text, options, () => {
-            if (ref.current?.toDataURL()) {
-                cache.set(text, ref.current?.toDataURL())
+        qr.toCanvas(ref.current, text, options, (err: Error) => {
+            if (err) throw err
```

Comment:
Replace QR Code area show a error report,
e.q: QRCode Render failed. (add to i18n)

Reply:
CC @neruthes need design a copywriting.
1. `This function is temporarily unavailable`
2. `QRCode Render failed`
do you have a better idea?

Reply:
I think that "Unknown error; please try again." will be good.

Reply:
OK, The function current is silent failure is a bad UX.
i think he can solve it.

Reply:
> I think that "Unknown error; please try again." will be good.

Click the text, try re-render the qrcode?

Reply:
> Click the text, try re-render the qrcode?

Making the "try again" part be clickable with certain styling (color & text-decoration underline) will be good. The user can click "try again" in the error message text line to try again.